app/models/custom_button.rb

```diff
@@ -139,7 +139,7 @@ def self.available_for_user(user, group)
   end
 
   def self.get_user(user)
-    user = User.in_region.find_by_userid(user) if user.kind_of?(String)
+    user = User.find_by_userid(user) if user.kind_of?(String)
```

Comment:
since users are replicated won't this change behavior:

On central region with region id 10 and this in the central's db:

```
1 joe
10000000000001 joe
```

Before:
`get_user("joe")`  => User(id 10000000000001)

After
`get_user("joe")`  => User(id 10000000000001) or User(id 1)